com.vysp3r.ProtonPlus.json

```diff
@@ -0,0 +1,33 @@
+{
+    "app-id" : "com.vysp3r.ProtonPlus",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "43",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "protonplus",
+    "only-arches": ["x86_64"],
```

Comment:
this doesn't do what you think it does. It's actually an error here.

Reply:
How can I test locally? When i run this inside GNOME Builder it works :/